app/models/event_item.rb

```diff
@@ -0,0 +1,10 @@
+class EventItem < ApplicationRecord
```

Comment:
We must use namespace like `Event::Item`.
Let's not pollute our /models folder.

Reply:
@alexshgov I done with Event::Item, but I have problem with dashboard. He try find EventItem dashboard and culdn't find him

Reply:
So I'm using the EventItem

Reply:
You actually can force administrate to understand a namespaced models.
```ruby
# initializers/administrate.rb

class EventItem < Event::Item; end
```

Reply:
@alexshgov From initializers its does not work, I am define this in dashboard for and appear this error [ undefined method `new_record?' for nil:NilClass ]. It happens when i am want create new event

Reply:
It may still be worth using the EventItem, because with Event :: Item, even the output does not work normally.